dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/Update.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.dependencies;
+
+import java.util.Objects;
+
+import org.eclipse.aether.artifact.Artifact;
+
+/**
+ * Holder for a suggested dependency update. 
+ */
+public class Update {
```

Comment:
Have you considered using [AutoValue](https://github.com/google/auto/tree/master/value)? It reduces a lot of boilerplate.

Reply:
Yes, and rejected it for multiple reasons including excess dependencies and idiosyncratic code.

Reply:
Also we need a specific toString method.